src/Brancho/Command/BranchBuilderCommand.php

```diff
@@ -60,7 +60,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
         $resolvedBranchNames = $brancho->resolveBranchNames($input, $output);
 
         if (!$resolvedBranchNames) {
-            $output->writeln('<fg=red>The resolved branch name is empty something went wrong.</>');
+            $output->writeln('<fg=red>No branches to create.</>');
```

Comment:
I chose a generic message because that will be also triggered if the user does not allow creating the parent-less branch creation.

If any error happens while getting data from Jira [it's already written on the output interface](https://github.com/spryker-sdk/brancho/blob/fdcd7df77a41bdca5362ee7283d82b034b7dd623/src/Brancho/Resolver/JiraResolver.php#L57) so I don't believe we need a second message saying "an error occurred", a generic one explaining that any branch was created does the job.